modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/DefinitionOfTermsContainer/DefinitionOfTermsContainer.tsx

```diff
@@ -20,6 +20,26 @@ interface DefinitionOfTermsContainerProps {
 	objectDefinitions: ObjectDefinition[];
 }
 
+function hasLocalizedField(
+	objectDefinitions: ObjectDefinition[],
+	entityId: number
+): boolean {
+	if (!objectDefinitions.length || !entityId) {
+		return false;
+	}
+
+	const selectedObjectDefinition =
+		objectDefinitionUtils.findObjectDefinitionById(
+			objectDefinitions,
+			entityId
+		);
+
+	return Boolean(
+		selectedObjectDefinition &&
+			fieldsUtils.hasLocalizedField(selectedObjectDefinition!)
```

Comment:
also to make things more readable, instead of using [the non-null assertion (!)](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator) you could put `selectedObjectDefinition as Type`, changing the `Type` for the respective type of the variable